Sources/BraintreeCore/BTCoreConstants.swift

```diff
@@ -1,29 +1,39 @@
 import Foundation
 
-/// :nodoc:
+/// :nodoc: This class is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
 @_documentation(visibility: private)
 @objcMembers public class BTCoreConstants: NSObject {
 
+    /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    @_documentation(visibility: private)
     public static var braintreeSDKVersion: String = "6.0.0-beta4"
 
-    public static var apiVersion: String = "2016-10-07"
-    
-    public static var graphQLVersion: String = "2018-03-06"
-    
+    /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    @_documentation(visibility: private)
```

Comment:
Do we need this for each property or is it covered by the class one?

Reply:
Yep, you are right, only needed at the class level. Updated in this commit https://github.com/braintree/braintree_ios/pull/1052/commits/90bd774f53b6ca5fc774abe6e86e7dd4e9027035 and double checked the compiled docs.